contracts/src/libraries/IGasRefunder.sol

```diff
@@ -13,23 +13,20 @@ interface IGasRefunder {
 }
 
 abstract contract GasRefundEnabled {
-    modifier refundsGasWithCalldata(IGasRefunder gasRefunder, address payable spender) {
+    /// @dev this refunds the sender for execution costs and for calldata as part of the tx input if `msg.sender == tx.origin`
```

Comment:
Slightly ambiguous reading of this comment - could be construed as there being no refund at all unless msg.sender == tx.origin